worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -1681,9 +1681,26 @@ public int makePyramid(BlockVector3 position, Pattern block, int size, boolean f
      * @param radius the radius
      * @return number of blocks affected
      * @throws MaxChangedBlocksException thrown if too many blocks are changed
+     * @deprecated Use {@link #thaw(BlockVector3, double, int)}.
      */
+    @Deprecated
     public int thaw(BlockVector3 position, double radius)
-            throws MaxChangedBlocksException {
+        throws MaxChangedBlocksException {
+        return thaw(position, radius,
+            WorldEdit.getInstance().getConfiguration().defaultVerticalHeight);
+    }
+
+    /**
+     * Thaw blocks in a cylinder.
+     *
+     * @param position the position
+     * @param radius the radius
+     * @param height the height (upwards and downwards)
+     * @return number of blocks affected
+     * @throws MaxChangedBlocksException thrown if too many blocks are changed
+     */
+    public int thaw(BlockVector3 position, double radius, int height)
```

Comment:
Wouldn't the height bar part of the radius?

Reply:
No, we're making these into cylinders. The height will usually be wildly different (i.e. 255) for most ops.